js/pages/configuration/configuration.js

```diff
@@ -130,6 +132,20 @@ define([
 			alert("Local Storage has been cleared.  Please refresh the page to reload configuration information.")
 		};
 
+		clearServerCache() {
+      cacheApi.clearCache().then(() => {
+        alert("Server cache has been cleared.");
+      });
+    };
+    
+    canClearServerCache() {
+      if (!config.userAuthenticationEnabled) {
+        return false;
+      } else {
+        return (config.userAuthenticationEnabled && this.isAuthenticated() && authApi.isPermittedClearServerCache());
```

Comment:
Does the simple `config.userAuthenticationEnabled && (this.isAuthenticated() && authApi.isPermittedClearServerCache())` not enough?

Reply:
agree, fixed